docs/debugging.md

```diff
@@ -0,0 +1,40 @@
+## Debugging Tutorial
+
+### For debugging purposes the following must be installed :
+
+[Visual Studio Code](https://code.visualstudio.com/)
+[Node.js](https://www.npmjs.com/) - to setup node.js you may follow the instructions in [Environment Setup](https://github.com/Seneca-CDOT/telescope/blob/master/docs/environment-setup.md)
+
+### Setting up VS Code and Launching Telescope :
+
+To begin debugging one must make sure that the telescope launch.json file can be found as VS code will automatically generate it's own launch.json file if none is found.
```

Comment:
This sentence is true, but we already have it in our repo, so it's not necessary.